runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java

```diff
@@ -218,6 +240,38 @@ private void onClanMemberLeft(ClanMemberLeft event)
 		getCallerList();
 	}
 
+	private void onPlayerSpawned(PlayerSpawned event)
+	{
+		final Player player = event.getPlayer();
+
+		if (player == client.getLocalPlayer())
+		{
+			return;
+		}
+
+		if (checkWildy() && isShowAgilityLevel())
+		{
+			if (!hiscoreCheckBlacklist.isEmpty() && hiscoreCheckBlacklist.keySet().contains(player.getName()))
+			{
+				return;
+			}
+
+			executorService.submit(() ->
+			{
+				try
+				{
+					hiscoreCheckBlacklist.put(player.getName(), client.getTickCount() + 6000); // TODO : Config? Handle clearing expired entries?
```

Comment:
move before the `try` or into a `finally` after the catch
use a `private static final ONE_HOUR = 6000;`